### ISSUE TITLE
add SearchParameter resource

### ISSUE THREAD

**Issue by abookin** [fhirbase/fhirbase#11]
I think, You should add SearchParameter resource to fhirbase schema. It would simplify the development of the search operation in application layer.

`CREATE TABLE IF NOT EXISTS "searchparameter" (id text primary key,  txid bigint not null,  ts timestamptz DEFAULT current_timestamp,  resource_type text default 'SearchParameter',  status resource_status not null,  resource jsonb not null); `

`CREATE TABLE IF NOT EXISTS "searchparameter_history" (  id text,  txid bigint not null,  ts timestamptz DEFAULT current_timestamp,  resource_type text default 'SearchParameter',  status resource_status not null,  resource jsonb not null,PRIMARY KEY (id, txid));`

**Comment by nishasumesh** [comment]
Entire search functionality is missing in fhirbase 3.0

**Comment by niquola** [comment]
yes, we will move it into documentation, because it’s very application specific - how to manage metadata. We are working on fhirpath extension, which will make it easy to implement FHIR search.

**Comment by Bolik** [comment]
I think, that searchParameter resource must exists, at least, for user-defined search parameters

**Comment by mlapshin** [comment]
You can manually create `searchparameter` and `searchparameter_history` table with the same schema as for any other resource table, it's not a big deal. CRUD functions should work for such custom resource.
